gitpod-io/gitpod
# [observability] Add OpenVSX proxy dashboard

## Description
Adds a Grafana dashboard for the OpenVSX proxy component.

Fixes https://github.com/gitpod-io/gitpod/issues/6229

## How to test

Visit https://grafana-clu-openvsx-dashboard.preview.gitpod-dev.com/d/HNOvmGpxgd/openvsx-proxy?orgId=1&refresh=5s to see the dashboard. Probably start some workspaces and/or search for extensions to get some data.

- Staging: https://grafana.gitpod-staging.com/d/HNOvmGpxgd/openvsx-proxy?orgId=1
- Production: https://grafana.gitpod.io/d/HNOvmGpxgd/openvsx-proxy?orgId=1

## Release Notes

```release-note
NONE
```
